client-java/instrumentation/pom.xml

```diff
@@ -34,6 +34,11 @@
             <artifactId>asm-commons</artifactId>
         </dependency>
 
+        <!--  handle sql command      -->
+        <dependency>
+            <groupId>com.github.jsqlparser</groupId>
+            <artifactId>jsqlparser</artifactId>
+        </dependency>
```

Comment:
if you add dependencies to `instrumentation` those must be shaded. see bottom of pom file for `maven-shade-plugin`

Reply:
thanks! updated